catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -150,10 +150,13 @@ public synchronized void failSource(String sourceId) {
         failedResponse.getProperties()
                 .put("elapsed", -1L);
 
-        updateStatus(sourceId, failedResponse);
+        updateStatus(sourceId, failedResponse, cause);
```

Comment:
Instead of passing the cause into `updateStatus` you should replace the `new Exception()` with `cause`.  The processing details is the only place `updateStatus` should read exceptions from for failure reasons.